src/pages/drawing/drawing.vue

```diff
@@ -114,14 +113,15 @@ export default {
       title
     }
   },
-  beforeDestroy () {
-      const data = ['my functionality id', true];
-      this.$root.channel.postRequest('DrawTools.StopDrawingRequest', data);
-      this.$root.channel.log('DrawTools.StopDrawingRequest posted with data:', data);
-  },
```

Comment:
Why is the remove the cleanup? This means that the draw functionality will remain active when moving between examples and might cause problems in other examples.

Reply:
Whoops. Mistake.